### ISSUE TITLE
Missing text for opinion_increased_crown_obligation

### ISSUE THREAD

**Issue by TerryDuke** [JonStryker/Witcher#102]
**Repro Steps**
1. Be a ruler as a long-lived race (not sure if this is required).
2. Issue an edit to increase crown obligation.
3. Look at the opinions of you from your vassals by hovering over the rating.

**Actual**
You see "opinion_increased_crown_obligation: -20"

**Expected**
That you see descriptive text instead of the variable name.

**Note**
It seems that in W_opinion_modifiers.csv the descriptive text is defined as:
 ```opinion_increased_crown_obligations;Increased Crown Obligations;;;;;;;;;;;;;x``` (with an 's')

Whereas in W_opinion_modifiers.txt and W_edict_decisions.txt it's referred to as ```opinion_increased_crown_obligation```.

Hence I will suggest that I create a fix that just removes the 's' from W_opinion-modifiers.csv file.

![bug21-opinion_increased_crown_obligation-small](https://user-images.githubusercontent.com/39809872/41571254-5bdc1f2e-7340-11e8-903c-5d828df60e0d.png)

**Addendum**
In text.log there are a lot of errors about missing descriptive text, and this is included.  In a different Issue we should probably fix all of them at once:

```
[opinion_modifier.cpp:519]: Reference to undefined loc key 'honest_neg_small'  file: common/opinion_modifiers/00_opinion_modifiers.txt line: 1152
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_my_chinese_pet'  file: common/opinion_modifiers/00_opinion_modifiers.txt line: 3990
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_duel_target'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 1
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_joust_target'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 6
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_battle_duel_target'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 11
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_jousting_target'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 16
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_attending_tournament'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 21
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_queen_love_beauty'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 24
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_queen_love_beauty_chosen'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 27
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_potential_tourny_rival'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 30
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_spectating_tournament'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 33
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_tournament_spectator'  file: common/opinion_modifiers/DE_opinion_modifiers.txt line: 36
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_increased_crown_obligation'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 9
[opinion_modifier.cpp:519]: Reference to undefined loc key 'tolerance_0_negative_10'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 27
[opinion_modifier.cpp:519]: Reference to undefined loc key 'tolerance_2_negative_20'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 30
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_raped_close_kin'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 80
[opinion_modifier.cpp:519]: Reference to undefined loc key 'flower_gift'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 128
[opinion_modifier.cpp:519]: Reference to undefined loc key 'inconsiderate_arse'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 131
[opinion_modifier.cpp:519]: Reference to undefined loc key 'shown_regret'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 134
[opinion_modifier.cpp:519]: Reference to undefined loc key 'respected_opinion'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 137
[opinion_modifier.cpp:519]: Reference to undefined loc key 'disrespected_costumes'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 140
[opinion_modifier.cpp:519]: Reference to undefined loc key 'laughed_off'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 143
[opinion_modifier.cpp:519]: Reference to undefined loc key 'rejected_proposition'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 146
[opinion_modifier.cpp:519]: Reference to undefined loc key 'sparring_partner'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 149
[opinion_modifier.cpp:519]: Reference to undefined loc key 'thrilling_horse_ride'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 152
[opinion_modifier.cpp:519]: Reference to undefined loc key 'careless_risks'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 155
[opinion_modifier.cpp:519]: Reference to undefined loc key 'fun_killer'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 158
[opinion_modifier.cpp:519]: Reference to undefined loc key 'opinion_asked_to_stay'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 215
[opinion_modifier.cpp:519]: Reference to undefined loc key 'hobbit_greatparty'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 231
[opinion_modifier.cpp:519]: Reference to undefined loc key 'hobbit_goodparty'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 234
[opinion_modifier.cpp:519]: Reference to undefined loc key 'hobbit_boringparty'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 237
[opinion_modifier.cpp:519]: Reference to undefined loc key 'hobbit_badparty'  file: common/opinion_modifiers/W_opinion_modifiers.txt line: 240
```

**Comment by TerryDuke** [comment]
Proposed fix in Pull Request #103

**Comment by rquinio** [comment]
About missing loc key errors, no need to fix the ones from vanilla files, i.e. not wtarting with W_ prefix (we don't even know if they are used in any event). The other ones could be in unfinished/disabled events, to be checked !
